src/calculateRentalCost.js

```diff
@@ -3,8 +3,20 @@
  *
  * @return {number}
  */
-function calculateRentalCost(days) {
-  // write code here
+function CalculateRentalCost(days) {
+  const dailyCost = 40;
+
+  let totalCost = days * dailyCost;
+
+  if (days >= 3) {
+    totalCost -= 20;
+  }
+
+  if (days >= 7) {
+    totalCost -= 30;
+  }
```

Comment:
The discount logic needs to be revised. If the number of days is 7 or more, only the $50 discount should apply, not both $20 and $30. Consider using an `else if` structure to ensure only one discount is applied.